lib/process-services/src/lib/process-list/services/process-filter.service.ts

```diff
@@ -72,8 +71,7 @@ export class ProcessFilterService {
     getProcessFilterById(filterId: number, appId?: number): Observable<FilterProcessRepresentationModel> {
         return from(this.callApiProcessFilters(appId))
             .pipe(
-                map((response: any) => response.data.find((filter) => filter.id === filterId)),
-                catchError((err) => this.handleProcessError(err))
+                map((response: any) => response.data.find((filter) => filter.id === filterId))
```

Comment:
We can remove `any` here right?

Reply:
removed and fixed typing issue discovered after that :)